docs/api-docs/storefront/scripts-overview.mdx

```diff
@@ -106,6 +106,26 @@ Solutions that add dynamic script support using revisions to or additions of scr
 
 Release notes can work for all storefronts. Changes to the [OAuth scopes](/docs/start/authentication/api-accounts#oauth-scopes) of apps that use [app-level API accounts](/docs/start/authentication/api-accounts#app-level-api-accounts) require the store owner or other authorized user to accept the new scopes, which could provide a mechanism to trigger release note display. Store owners aren't prompted to reauthorize until they next open an app in the control panel, so direct contact may be advisable from a conversion perspective. You can't change the OAuth scopes for a [store-level API account](/docs/start/authentication/api-accounts#store-level-api-accounts), so directly contacting the store or stores that you want to change may be your only option for disclosure.
 
+## Subresource integrity
+
+[Subresource integrity (SRI)](https://developer.mozilla.org/en-US/docs/Web/Security/Subresource_Integrity) is a security feature browsers use to verify that attackers have not manipulated the script. 
```

Comment:
SRI can be used to verify any kind of external hosted resource. Aside from scripts, it can also be used to check the integrity of CSS stylesheets, images, etc. Minor nitpick since PCI 4.0 only covers scripts, but not sure if we want to be broad in our definition of SRI here.

Reply:
Thanks, I just rephrased slightly to mention that SRI is for external hosted resources